src/config.rs

```diff
@@ -64,7 +65,13 @@ pub fn load() -> Result<Config, Box<dyn std::error::Error>>{
     };
 
     let file_content = std::fs::read_to_string(path).unwrap();
-    let yaml = &yaml_rust::YamlLoader::load_from_str(&file_content).unwrap()[0];
+
+    let vec_yaml = yaml_rust::YamlLoader::load_from_str(&file_content).unwrap();
+    let mut yaml = &Yaml::Null;

```

Comment:
Overall I'd like to keep the style the same so either every `yaml_rust::......` usage has to change or we rewrite it to use namespaces

```suggestion
    let mut yaml = &yaml_rust::Yaml::Null;
```